src/libs/antares/study/include/antares/study/parts/thermal/cluster_list.h

```diff
@@ -94,6 +94,11 @@ class ThermalClusterList: public ClusterList<ThermalCluster>
                | std::views::filter(std::not_fn(&ThermalCluster::isMustRun));
     }
 
+    auto each_has_min_gen() const
+    {
+        return each_enabled_and_not_mustrun() | std::views::filter(&ThermalCluster::hasMinGen);
+    }
```

Comment:
Probably not necessary

Reply:
I removed `hasMinGenCount()` from class **ThermalClusterList**.
But for this one, what do you suggest ?

Reply:
I suggest not using this filter since it's extremely unusual for output variables to be printed based on such a condition. Print the value for every enabled cluster.

Reply:
You mean that cluster is a business concept, and` each_has_min_gen()` unnecessarily bloats class **ThermalClusterList** with a function used only in output ?

Reply:
I think we should produce this variable for all enabled cluster, no matter if they have a min-gen or not